### PR TITLE
doc/md/guides/mysql: Add article about prefix indexes

### PR DESCRIPTION
**Pull request by priteshtm**

### CODE REVIEW

File: doc/md/guides/mysql/prefix-indexes.md [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+  `id` mediumint(8) unsigned NOT NULL auto_increment,
+  `sender` varchar(255),
+  `receiver` varchar(255),
+  `sender_address` varchar(255),
+  `email` varchar(255),
+  `cryptic_message` TEXT,
+  `suspicious` BOOLEAN,
+  `sent_date` varchar(255),
+  PRIMARY KEY (`id`)
+) AUTO_INCREMENT=1;
```
Comment (a8m) [inline doc/md/guides/mysql/prefix-indexes.md:57]:
```suggestion
  PRIMARY KEY (`id`)
);
```

File: doc/md/guides/mysql/prefix-indexes.md [hunk 2]
```diff
@@ -0,0 +1,451 @@
...
+
+The storage space used by the index is as following:
+
+```sql
+SELECT 
+stat_value
+FROM 
+mysql.innodb_index_stats 
+WHERE 
+index_name = 'email_idx' AND stat_name= 'size';
```
Comment (a8m) [inline doc/md/guides/mysql/prefix-indexes.md:262]: indent.

File: doc/md/guides/mysql/prefix-indexes.md [hunk 3]
```diff
@@ -0,0 +1,451 @@
...
+:::
+
+For our example, keeping `innodb_page_size` as 16KB in mind, our index uses ~32MB disk space. Now, we can further improve the query performance as well as reduce the storage used by the index by using a prefix index on the email column. Let’s create a prefix index with the following command:
+
+```sql
+ALTER TABLE fbi_suspects DROP INDEX email_idx;
+CREATE INDEX 
+    email_prefix_idx 
+ON 
+    fbi_suspects(email(5));
```
Comment (a8m) [inline doc/md/guides/mysql/prefix-indexes.md:286]:
```suggestion
ALTER TABLE fbi_suspects DROP INDEX email_idx;
CREATE INDEX 
    email_prefix_idx 
ON 
    fbi_suspects(email(5));
```

ALTER can have both ADD and DROP. Maybe stay consistent and use one ALTER or DROP and CREATE.

File: doc/md/guides/mysql/prefix-indexes.md [hunk 4]
```diff
@@ -0,0 +1,451 @@
...
+
+And the storage space used by the index is as follows:
+
+```sql
+SELECT 
+stat_value
+FROM 
+mysql.innodb_index_stats 
+WHERE 
+index_name = 'email_prefix_idx' AND stat_name= 'size';
```
Comment (a8m) [inline doc/md/guides/mysql/prefix-indexes.md:314]: indentation

File: doc/md/guides/mysql/prefix-indexes.md [hunk 5]
```diff
@@ -0,0 +1,451 @@
...
+Use the arrow keys to navigate: ↓ ↑ → ← 
+? Are you sure?: 
+  ▸ Apply
+    Abort
+```
+
+To verify that our new index was created, open the database command line tool from the previous step and run:
+
+```sql
+SHOW INDEXES FROM scorecard;
```
[the remark this answers is not in the document]
Comment (priteshtm) [reply doc/md/guides/mysql/prefix-indexes.md]: I have used `atlas schema inspect -u "mysql://root:password@localhost:3306/fbi_suspects" | grep -A5 index`, is that good?

**Review by a8m — APPROVED** [review]

**Pull request merged**